sos/cleaner/__init__.py

```diff
@@ -92,6 +95,17 @@ def __init__(self, parser=None, args=None, cmdline=None, in_place=False,
             SoSUsernameParser(self.cleaner_mapping, self.opts.usernames)
         ]
 
+        self.archive_types = [
+            SoSDirectoryArchive,
```

Comment:
Just a nitpick: following the naming logic for `SoSCollector[Directory|Archive]`, does not have sense to name this `SoSReportDirectory`?

Reply:
I'm trying to see if I can merge this with `SoSCollectDirectory`, but if I can't then ack to the rename